app/test_main.py

```diff
@@ -1,3 +1,18 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected",
+        [
+            (0, 0, [0, 0]),
+            (14, 14, [0, 0]),
+            (15, 15, [1, 1]),
+            (23, 23, [1, 1]),
+            (24, 24, [2, 2]),
+            (27, 27, [2, 2]),
+            (28, 28, [3, 2]),
+            (100, 100, [21, 17])
+        ]
+    )
```

Comment:
Consider adding test cases for invalid inputs, such as negative numbers or non-integer values, to ensure that the `get_human_age` function handles these cases correctly. This aligns with the checklist point to test edge situations and ensure the function raises the correct exceptions for incorrect data types.